test/docker-compose.yml

```diff
@@ -23,3 +23,19 @@ services:
       dockerfile: Dockerfile
       context: .
     command: /app/print-random-stuff.sh
+
+  my-service4:
+    build:
+      dockerfile: Dockerfile
+      context: .
+    command: /app/print-random-stuff.sh
+    profiles:
+      - foo
+
+  my-service5:
+    build:
+      dockerfile: Dockerfile
+      context: .
+    command: /app/print-random-stuff.sh
+    profiles:
+      - bar
```

Comment:
Not really sure how this compose file is used as part of automated testing. I'm hoping a pipeline will tell me off otherwise I'll just assume it's used for manual testing.

Reply:
fyi - to manually test i used the following command:

`lazydocker -f test/docker-compose.yml -p foo --profile bar`